src/test/ui/did_you_mean/issue-54109-and_instead_of_ampersands.stderr

```diff
@@ -1,58 +1,75 @@
-error: expected `{`, found `and`
-  --> $DIR/issue-54109-and_instead_of_ampersands.rs:4:10
+error: `and` is not a logical operator
+  --> $DIR/issue-54109-and_instead_of_ampersands.rs:7:15
+   |
+LL |     let _ = a and b;
+   |               ^^^ help: use `&&` to perform logical conjunction
+   |
+   = note: unlike in e.g., python and PHP, `&&` and `||` are used for logical operators
```

Comment:
I don't know if the note calling out python and PHP here is a good idea.

Reply:
I debated this with myself a few times but in the end I thought that habits from python (PHP less so) would be the most likely source of using `and` so it felt like relevant information for such users.